qcodes/instrument/parameter.py

```diff
@@ -371,8 +372,8 @@ def snapshot_base(self, update: bool = True,
                       ) -> Dict:
         """
         State of the parameter as a JSON-compatible dict (everything that
-        the custom JSON encoder class :class:'qcodes.utils.helpers.NumpyJSONEncoder'
-        supports).
+        the custom JSON encoder class
+        :class:'qcodes.utils.helpers.NumpyJSONEncoder' supports).
```

Comment:
unrelated to this PR, but:
```suggestion
        :class:`qcodes.utils.helpers.NumpyJSONEncoder` supports).
```